### PR TITLE
Fix for #162 - DefaultConverter doesn't support array serialisation

### PR DESCRIPTION
**Pull request by alfonso-eusebio**

### CODE REVIEW

File: src/main/kotlin/com/beust/klaxon/DefaultConverter.kt [hunk 1]
```diff
@@ -44,6 +44,10 @@ class DefaultConverter(private val klaxon: Klaxon, private val allPaths: HashMap
         val result = when (value) {
             is String, is Enum<*> -> "\"" + Render.escapeString(value.toString()) + "\""
             is Double, is Float, is Int, is Boolean, is Long -> value.toString()
+            is Array<*> -> {
+                val elements = value.filterNotNull().map { klaxon.toJsonString(it) }
```
Comment (LaurenceGA) [inline src/main/kotlin/com/beust/klaxon/DefaultConverter.kt]: If I understand this correctly, when serializing an array, null elements aren't printed?
Why is this done?
JSON arrays can have null objects.
Serializing "listOf(1, 2, null, null, 3)" should produce "[1, 2, null, null, 3]", right?

Comment (LaurenceGA) [reply src/main/kotlin/com/beust/klaxon/DefaultConverter.kt]: ... Same goes for Collection down below

**Comment by alfonso-eusebio** [comment]
This change to support Arrays was implemented following the same principles applied to the existing logic for Collections, hence both ignore nulls.
Yours might be a valid point, but I think it should be discussed in a wider context, rather than within this particular merge request.

**Comment by LaurenceGA** [comment]
Yes, that makes sense. Sorry, just mentioned it when I saw it.
I already created a [an issue](https://github.com/cbeust/klaxon/issues/173) about it.

**Pull request merged**